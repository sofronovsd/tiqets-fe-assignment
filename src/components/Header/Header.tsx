import React from 'react';
import styled from "styled-components";

const StyledHeader = styled.header`
  display: flex;
  align-items: center;
  justify-content: center;
  align-self: stretch;
  padding: 64px 16px;
  background-color: #4E2870;
`

const Title = styled.h1`
  color: #FFF;
  font-family: Roboto Mono;
  font-size: 32px;
  font-style: normal;
  font-weight: 500;
  line-height: 40px;
  margin: 0;
`

const Header = () => {
  return (
    <StyledHeader>
      <Title>Plan your trip!</Title>
    </StyledHeader>
  );
};

export default Header;
