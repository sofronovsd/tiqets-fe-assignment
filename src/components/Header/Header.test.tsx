import {render} from "@testing-library/react";
import Header from "./Header";

describe('Header', () => {
  test('render', () => {
    const { container} = render(<Header />)
    expect(container).toMatchSnapshot()
  })
})
