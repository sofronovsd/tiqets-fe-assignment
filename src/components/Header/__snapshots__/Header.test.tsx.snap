// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header render 1`] = `
<div>
  <header
    class="sc-bdfCDU eamTCr"
  >
    <h1
      class="sc-gsTDqH jOfYXJ"
    >
      Plan your trip!
    </h1>
  </header>
</div>
`;
